### ISSUE TITLE
LSTM with character-level embeddings from FastText. Dimensionality issue. 

### ISSUE THREAD

**Issue by saroufimc1** [keras-team/keras#9191]
Hi, 
I am having a dimensionality issue if I want to fit an LSTM models with pre-trained embeddings of words and subwords coming from FastText. 

e.g. (Assume I properly padded). 
I have an embedding matrix W of shape 10x100 where 10 is the dictionary size.

X = [[[1,2,3], [4,5,7], [3,2,1]],    #sentence 1. 1 sub-list = 1 word = indexes to be mapped to an embedding matrix
     [[9,3,1], [5,6,7], [1,8,4]]]    #sentence 2.

So X has a 3 dimensional shape:
- 1 dimension for number of sentences. 
- 1 dimension for the list of indices in 1 word (number of subwords for each word).
- 1 dimension for the number of words in 1 sentence. 

y = [[0,1], [1,0], [0,1],             # 1 label for each word of each sentence.
     [1,0], [0,1], [0,1]]

Using the  Embedding layer with my pre-trained model returns a dimensionality error when I call fit. 
How to deal with this issue? Should I just feed the embeddings directly as input? 
Please note that I want to be able to have batch sizes > 1 so I do not want to feed the LSTM with 1 sentence at a time as a workaround for my dimensionality issue... 

Thanks for your help!
